ora2/jobs/AddCourseToSandbox.groovy

```diff
@@ -0,0 +1,56 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    * folder-name: some-folder
+    * credential : your-ssh-credential-name
+    * email : e-mail-contact@example.com
+*/
+
+package ora2.jobs
+
+import org.yaml.snakeyaml.Yaml
+import org.yaml.snakeyaml.error.YAMLException
+
+class AddCourseToSandbox {
+
+    public static def job = { dslFactory, extraVars ->
+        return dslFactory.job(extraVars.get('folder-name', 'ora2') + '/add-course-to-sandbox') {
+                keepDependencies(false)
+                properties {
+                    rebuild {
+                        autoRebuild(false)
+                    }
+                    parameters {
+                        stringParam(
+                            'SANDBOX_BASE',
+                            'ora2',
+                            "For example, when the sandbox is foo.sandbox.edx.org, the sandbox_base is 'foo'"
+                        )
+
+                    }
+                    throttleJobProperty {
+                        maxConcurrentPerNode(0)
+                        maxConcurrentTotal(0)
+                        throttleEnabled(false)
+                        throttleOption('project')
+                        limitOneJobWithMatchingParams(false)
+                        paramsToUseForLimit('')
+                        matrixOptions {
+                            throttleMatrixBuilds(false)
+                            throttleMatrixConfigurations(false)
+                        }
+                    }
+                }
+                steps {
+                    shell(dslFactory.readFileFromWorkspace('ora2/resources/ssh-install-test-course.sh'))
+                }
+                publishers {
+                    mailer(extraVars['email'], false, false)
```

Comment:
Will double check with @cahrens, but yes, it looks like "dontNotifyEveryUnstableBuild" should be true, because we don't want to do that, since that's been a major complaint about the ora2 jobs that we've had.